.tool-versions-e

```diff
@@ -0,0 +1,2 @@
+elixir 1.6.0
+erlang 18.3
```

Comment:
Yeah, I don't know what this is, but I always get this file when I use asdf. Is it not normal?

I usually try to avoid checking it in, but I'll delete it now.

Reply:
I don't usually get that file :thinking: 

You could add it to your global git ignore